docs/statistics.md

```diff
@@ -111,11 +119,31 @@ The time elapsed, in milliseconds, since the SRT socket has been created (after
 
 ### pktSentTotal
 
-The total number of sent data packets, including retransmitted packets. Available for sender.
+The total number of sent DATA packets, including retransmitted packets ([pktRetransTotal](#pktRetransTotal)). Available for sender.
```

Comment:
Per mechanism explanation: "This metric is counted for every DATA packet that is sent over the network".

Reply:
I'll not add any additional comments here. I guess it's clear for the reader.

Reply:
The problem here is that the word "sent" is ambiguous.

"Sent" can be a packet from the application to SRT connection, or a UDP packet carrying the SRT data packet over the network. This should be clarified.